pkg/client/interface.go

```diff
@@ -12,6 +12,7 @@ type StackAPIClient interface {
 	StackCreate(ctx context.Context, stack types.StackCreate, options types.StackCreateOptions) (types.StackCreateResponse, error)
 	StackInspect(ctx context.Context, id string) (types.Stack, error)
 	StackList(ctx context.Context, options types.StackListOptions) ([]types.Stack, error)
+	//StackUpdate(ctx context.Context, id string, version types.Version, stack types.StackCreate, options types.StackUpdateOptions) error
```

Comment:
The update model would either use StackCreate or something along those lines instead of the full spec object